UPGRADE-3.4.md

```diff
@@ -0,0 +1,10 @@
+UPGRADE FROM 3.3 to 3.4
+=======================
+
+FrameworkBundle
+---------------
+
+ * Using `KERNEL_DIR` or the automatic guessing based on the `phpunit.xml` file
+   location is deprecated since 3.4. Either set `KERNEL_CLASS` in your `phpunit.xml` 
+   to the fully-qualified class name of your Kernel or override the `KernelTestCase::createKernel()` 
+   method. Not setting `KERNEL_CLASS` will throw an exception on 4.0.
```

Comment:
What about:

```markdown
Using `KERNEL_DIR` or the automatic guessing based on the `phpunit.xml` file
location is deprecated since 3.4. Set `KERNEL_CLASS` to the fully-qualified
class name of your Kernel instead. Not setting `KERNEL_CLASS` will throw an
exception on 4.0 unless you override `KernelTestCase::createKernel()` or
`KernelTestCase::getKernelClass()` method.
```